CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ and this project adheres to
   - [#3060](https://github.com/bpftrace/bpftrace/pull/3060)
 - Disable func builtin for kretprobes and uretprobes when `get_func_ip` feature is not available
   - [#2645](https://github.com/bpftrace/bpftrace/pull/2645)
+- Attach probe after function prologue and probe inlined functions
```

Comment:
Perhaps better to have 2 changelog entries (both pointing to same PR).

These are big features (user PoV)!